gdc_ng_models/models/batch.py

```diff
@@ -0,0 +1,131 @@
+import json
+import sqlalchemy
+from sqlalchemy import orm
+from sqlalchemy.ext import declarative
+
+Base = declarative.declarative_base()
+
+
+class Batch(Base):
+    """Batch class to represent a collection of nodes.
+
+    Attributes:
+        id: unique identifier for the batch
+        name: name given to the batch
+        description: optional description for the batch
+        project_id: project the batch is a part of
+        created_datetime: the date and time when the batch is created
+    """
+
+    __tablename__ = "batch"
+    __mapper_args__ = {"eager_defaults": True}
+
+    # start sequence at high number to avoid collisions with existing batch_id
+    id_seq = sqlalchemy.Sequence("batch_id_seq", metadata=Base.metadata, start=1000)
+
+    id = sqlalchemy.Column(
+        sqlalchemy.BigInteger,
+        nullable=False,
+        primary_key=True,
```

Comment:
Last year, we changed our preference on how to create indices.  (I can't remember if we wrote this down. So, I should do that.) Instead of using `primary_key`, we can use`PrimaryKeyConstraint`, which allows us to define our index names. For an example, see:
https://github.com/NCI-GDC/gdc-ng-models/blob/master/gdc_ng_models/models/studyrule.py#L30-L33

For primary keys, the naming convention is `<table_name>_pk`.